libs/source/Socket.cpp

```diff
@@ -61,7 +61,6 @@ int Socket::setNOSIGPIPE() {
     int ret = setSockOpt(SO_NOSIGPIPE, &on, sizeof(on));
     if (ret != 0) {
         LOG(IPC, Sev::Warning, "Cannot set SO_NOSIGPIPE for socket");
-        perror("setsockopt():");
```

Comment:
The same as above — isn't this information possibly helpful?

Reply:
in this case, yes.